.circleci/config.yml

```diff
@@ -43,7 +43,7 @@ workflows:
     jobs:
       - build:
           <<: *common_filters
-      - unit_test:
```

Comment:
Hmmm, miscommunication I think. I had in mind two possibilities.
Using the image as a guide.

![Screenshot from 2021-04-13 11-58-18](https://user-images.githubusercontent.com/1730679/114583527-ad5f8580-9c4f-11eb-8703-b48df40e9664.png)

a) Splitting out the non-confidential_cwlrunner tests into a fourth job on the right (I dislike having the unit tests combined with non unit tests) Additionally, the final stuff like building and publishing the Docker container seems unrelated too and should probably be dependent on all four jobs on the right anyway. Maybe they should be a final job. 
b) Moving the actual unit tests to the build step on the left and just leaving the cwlrunner tests here 

That said, this has absolutely nothing to do with hibernate or pg, so can just be new ticket rather than holding up this ticket.

Reply:
I'd thought of that, but I assumed there's overhead to additional jobs, and I was trying to avoid that.

I'll go ahead and rename the step back to unit_tests, to avoid less confusion and need for changing checks, then create a ticket for separation.

Reply:
I think we can live with the overhead since we're no longer on Travis-CI. 
Besides, running unit tests in the build step might rule out some obvious failures and skip the four on the right.